scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+## parameters
+CONTRACT_ADDRESS='link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8'
+FROM_ACCOUNT='link146asaycmtydq45kxc8evntqfgepagygelel00h'
+URL='http://localhost:26658'
+CHAIN_OPTION='--chain-id simd-testing --keyring-backend test -b block -o json -y'
+
+## Wait until the height of the block is greater than or equal to 1.
+timeout 60 bash -c "until fnsad query block 1 --node ${URL} --chain-id simd-testing > /dev/null; do sleep 0.5; done"
+# If the timeout fails, the process will be terminated abnormally.
+exitstatus=$?
+if [[ $exitstatus -ne 0 ]]; then
+    echo "waiting finschia node is timeout"
+    exit 1
+fi
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+# This is a function that checks if executeMsg is successful.
+check_run_info() {
+  local res="$1"
+  local msg="$2"
+	if [[ "$res" == *"failed"* ]]; then
+		echo -e "$msg$res"
+		exit 1
+	fi
+}
+
+#*** store collection contract ***
+result=$(fnsad tx wasm store ${artifacts_path}/collection.wasm --node ${URL} --from ${FROM_ACCOUNT} --gas 10000000 ${CHAIN_OPTION})
+
+## confirm a result of `store`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "store: "
+
+#*** instantiate collection contract ***
+result=$(fnsad tx wasm instantiate 1 '{"name":"collection_name","uri":"collection_uri","meta":"collection_meta", "owner":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"}' --label collection1 --admin link146asaycmtydq45kxc8evntqfgepagygelel00h  --from link146asaycmtydq45kxc8evntqfgepagygelel00h --node ${URL} ${CHAIN_OPTION})
+
+## confirm a result of `instantiate`
+raw_log=$(echo ${result} | jq .raw_log)
+check_run_info "${raw_log}" "instantiate: "
+
+#*** issue_nft collection contract ***
+result=$(fnsad tx wasm execute ${CONTRACT_ADDRESS} '{"issue_nft":{"name":"nft1_name","meta":"nft1_meta","owner":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"}}' --from ${FROM_ACCOUNT} --node ${URL} --gas 10000000 ${CHAIN_OPTION})
```

Comment:
ditto

Reply:
The contents of json are difficult to expand into variables. It will be difficult to return it, so please leave it as is.

Reply:
fixed here
[fix: change address to variable specification instead of hard code in…](https://github.com/Finschia/finschia-wasm/pull/3/commits/409bc9e4f57487615926518589c1c78926607913)